app/components/Swapbox.tsx

```diff
@@ -230,6 +240,28 @@ export const Swapbox = ({ fixedProductMarketMaker }: SwapboxProps) => {
     !!connectorChainId &&
     !supportedChains.some(supportedChain => supportedChain.id === connectorChainId);
 
+  const priceInUSD = collateralTokenUSDPrice
+    ? isBuying
+      ? formatValueWithFixedDecimals(collateralTokenUSDPrice, 2)
+      : formatValueWithFixedDecimals(
+          collateralTokenUSDPrice * +currentState.tokenPrice,
+          2
+        )
+    : null;
```

Comment:
Minor, non-blocking:
Maybe we can simplify a bit the ternary operator nesting, WDYT? 
```suggestion
  const priceInUSD = collateralTokenUSDPrice
    ? formatValueWithFixedDecimals(
        collateralTokenUSDPrice * (isBuying ? 1 : +currentState.tokenPrice),
        2
      )
    : null;
```

Reply:
I think this is harder to read. We are mixing code with math so you don't have a easy time understanding what the calculation is. But will create a `formattedPriceInUSD` variable to simplify reading.